database/db.go

```diff
@@ -17,28 +19,50 @@ type dbConfig struct {
 	password string
 }
 
-var config = dbConfig{
-	os.Getenv("POSTGRES_HOST"),
-	os.Getenv("POSTGRES_PORT"),
-	os.Getenv("POSTGRES_USER"),
-	os.Getenv("POSTGRES_DB"),
-	os.Getenv("POSTGRES_PASSWORD"),
+func getConfig() dbConfig {
+	return dbConfig{
+		os.Getenv("POSTGRES_HOST"),
+		os.Getenv("POSTGRES_PORT"),
+		os.Getenv("POSTGRES_USER"),
+		os.Getenv("POSTGRES_DB"),
+		os.Getenv("POSTGRES_PASSWORD"),
+	}
 }
 
-var dns = fmt.Sprintf(
-	"host=%s port=%s user=%s dbname=%s password=%s sslmode=disable",
-	config.host, config.port, config.user, config.dbname, config.password)
+func getTestConfig() dbConfig {
+	return dbConfig{
+		os.Getenv("POSTGRES_HOST"),
+		os.Getenv("POSTGRES_TEST_PORT"),
+		os.Getenv("POSTGRES_USER"),
+		os.Getenv("POSTGRES_TEST_DB"),
+		os.Getenv("POSTGRES_PASSWORD"),
+	}
+}
```

Comment:
Why are the tests in this file and not a db_test.go?

Reply:
So, this isn't code for testing the DB, but for other files. Namely, the `user` package tests. But yeah this is a little confusing.

Reply:
Not in love with that, but it is what it is 😆